packages/nx/src/command-line/release/config/config.ts

```diff
@@ -355,8 +356,16 @@ export async function createNxReleaseConfig(
     );
 
   // these options are not supported at the group level, only the root/command level
-  const rootVersionWithoutGlobalOptions = { ...rootVersionConfig };
+  const rootVersionWithoutGlobalOptions = {
```

Comment:
You left this underneath the comment saying it is not something which is supported at the group level, which doesn't make sense, but more broadly why are you creating a relationship between preVersionCommand and groupPreVersionCommand?

They are separate things, right now the way you have things it looks like if you set a preVersionCommand it will be run twice, once as a preVersionCommand and once as a groupPreVersionCommand?

Please remove this relationship